cocotb/share/makefiles/simulators/Makefile.aldec

```diff
@@ -63,8 +63,8 @@ ASIM_ARGS += $(SIM_ARGS)
 # Plusargs need to be passed to ASIM command not vsimsa
 ASIM_ARGS += $(PLUSARGS)
 
-RTL_LIBRARY ?= work
-ALOG_ARGS += +define+COCOTB_SIM -dbg -pli libgpi
+RTL_LIBRARY ?= $(SIM_BUILD)/work
+ALOG_ARGS += +define+COCOTB_SIM -dbg
```

Comment:
Removing `-pli libgpi` seems out of scope for this patch. Was it necessary? Can you put it in a separate commit?